### ISSUE TITLE
Causes accordion panel to no longer push down content

### ISSUE THREAD

**Issue by mickwieland** [j26design/SEO-for-Webpages#10]
When I place either the SEO for Webpages widget (v. 7.0) or the Book SEO widget (v. 5.2), it causes an accordion panel on the page to no longer push down content below it. When I remove both widgets, the accordion panel works correctly again. I've submitted my Muse file to Adobe Support and they confirmed that the issue is caused by both of these widgets. 

Can you confirm and possibly issue a fix? See my file. Issue is on home page, click on the "LEARN MORE" button towards bottom of page.
[The Internationalists 05.muse.zip](https://github.com/j26design/SEO-for-Webpages/files/1032298/The.Internationalists.05.muse.zip)

**Comment by jp26jp** [comment]
I'm trying to find a way to still do what you're trying to do, but this is a Muse bug. The widget shouldn't be doing this so the way that Muse is insert the html into the page is the problem.

**Comment by jp26jp** [comment]
Also, 2 essential widgets cannot be used on a page. That's just an FYI, not necessarily helpful for the bug.

**Comment by jp26jp** [comment]
Actually, this is somehow happening because the embedded html isn't pinned. If you pin the embedded html, it works as expected.

**Comment by mickwieland** [comment]
Hi John,

I assume you're referring to the embedded html for addthis.com? That's the only embedded html I'm using. I've pinned it as you suggest but that does not seem to correct the issue.

If you are referring to something else can you please clarify?


________________________________

**Comment by mickwieland** [comment]
P.S. - I removed the Book SEO widget v5.2 and replaced it with your old Book SEO widget  from 2014 and the issue went away. Then, when I went back to the new Book SEO widget, the issue returned.


________________________________

**Comment by jp26jp** [comment]
https://www.youtube.com/watch?v=dlJgvVVps1s

**Comment by jp26jp** [comment]
Also, I'm almost done with a much better SEO for Books widget that you'll be able to use.

**Comment by mickwieland** [comment]
Thanks for putting that video together. The fix does not, however, work consistently across all breakpoints.

I see that when I pin the embedded code, it works for the break point I’m viewing, but if you then look at all other breakpoints, the accordion does not push down consistently.

Only removing the SEO widget fixes the bug across all breakpoints. I’ve since re-worked my page to work around the issue, but you may want to put in some notes to Adobe about this bug in relation to your SEO widget.

Thanks,
Mick

**Comment by jp26jp** [comment]
Damn. Do you mind if I upload the file to Muse?

**Comment by jp26jp** [comment]
Wait, did you pin the HTML in every breakpoint?

**Comment by mickwieland** [comment]
Yes, pinned in all break points. Still no dice. Fine to upload file.

On Jun 7, 2017, at 8:23 PM, John R Perry <notifications@github.com<mailto:notifications@github.com>> wrote:


Wait, did you pin the HTML in every breakpoint?

**Comment by jp26jp** [comment]
[I posted the bug here](https://www.adobeprerelease.com/beta/FB228BCF-1DAD-475B-B5DF-678F6347DBED/feedback/item/MU-4198274)
